distribution/containers/Dockerfile.release

```diff
@@ -21,8 +21,12 @@ RUN adduser -h /home/download -s /sbin/nologin --disabled-password download
 # Run as "download" user
 USER download
 
+ENV HOME=/home/download
```

Comment:
For non-distroless (normal) images, it is unnecessary (and harmful) to override HOME inside the Dockerfile.

Reply:
as there is one by default.

Reply:
Updated